website/docs/installation/docker.mdx

```diff
@@ -23,5 +23,11 @@ import TabItem from '@theme/TabItem';
   docker run -it -p 8080:8080 -v $HOME/.tabby:/data tabbyml/tabby serve --model TabbyML/StarCoder-1B
   ```
 
+  For `arm64` CPUs, please clone the repository to your machine, then run these commands inside the repository:
+  ```bash title="run.sh"
+  docker build -t tabby:latest --build-arg ARCHTARGET=amd64 .
```

Comment:
should this be `arm64`?

Reply:
Yeah, I believe so from this [line](https://github.com/TabbyML/tabby/pull/1022/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R65)

Reply:
Note that the **`TARGETARCH`** ARG variable is set **automatically** by Docker's BuildKit. See: https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope